mbed/mbed.py

```diff
@@ -2752,7 +2761,7 @@ def target_(name=None, global_cfg=False, supported=False):
     dict(name='name', nargs='?', help='Default toolchain name. Example: ARM, GCC_ARM, IAR'),
     dict(name=['-G', '--global'], dest='global_cfg', action='store_true', help='Use global settings, not local'),
     dict(name=['-S', '--supported'], dest='supported', action='store_true', help='Shows supported matrix of targets and toolchains'),
-    help='Set or get default toolchain\n\n',
+    help='Set or get default toolchain\n',
```

Comment:
Could you move this unrelated change to another PR?

Reply:
I can move it to a separate PR, but I think it's related? It changes the cosmetics of the `mbed help` output, it means that target, toolchain and profile are grouped together.

Without it the output would be

        config     Tool configuration
        target     Set or get default target 
        toolchain  Set or get default toolchain
        
        profile    Set or get default profile(s)

Reply:
:+1: Thanks for the explanation.